pkg/operator/reporting/templates_test.go

```diff
@@ -678,9 +711,11 @@ func newTestReportQuery(testReportName, testNamespace, query string, inputs []me
 	}
 }
 
-func newTestInputDefinition(inputType string) metering.ReportQueryInputDefinition {
+func newTestInputDefinition(inputName, inputType string) metering.ReportQueryInputDefinition {
 	return metering.ReportQueryInputDefinition{
+		Name:     inputName,
 		Type:     inputType,
 		Required: true,
+		Default:  newDefault(`"` + inputName + `"`),
```

Comment:
We will want to test with an empty default, so this might not make sense. I'm not sure this function even makes perfect sense at this point